src/Authorization.AspNetCore/AuthorizationError.cs

```diff
@@ -26,7 +25,7 @@ public AuthorizationError(INode node, ValidationContext context, OperationType?
         /// Initializes a new instance of the <see cref="AuthorizationError"/> class for a specified authorization result with a specific error message.
         /// </summary>
         public AuthorizationError(INode node, ValidationContext context, string message, AuthorizationResult result)
-            : base(context.OriginalQuery, "6.1.1", message, node == null ? Array.Empty<INode>() : new INode[] { node })
```

Comment:
Why?

`   var s = new AuthorizationError(null, new Validation.ValidationContext { Document = new Language.AST.Document { OriginalQuery = "" } }, "", Microsoft.AspNetCore.Authorization.AuthorizationResult.Failed());
`

![изображение](https://user-images.githubusercontent.com/21261007/111539881-79dc2900-877f-11eb-8fea-12a76c8c6ca9.png)

![изображение](https://user-images.githubusercontent.com/21261007/111539927-8791ae80-877f-11eb-9d5b-b8dd880a89a3.png)

Reply:
`params` wraps `null` into `new INode[] { null }`

Reply:
See #470

Reply:
https://github.com/graphql-dotnet/server/pull/470#discussion_r537030726

Reply:
It didn't compile and intellisense said it couldn't cast `INode[]` to `INode`.  So I removed the array.  🤷‍♂️ Maybe intellisense picked up on the wrong overload; I dunno.

Reply:
I suggest we add additional overloads to the ValidationError constructor with a single INode so we don't have to write this hack in here.  Chances are that almost all calls only have a single node anyway, so it would reduce an allocation also.

Reply:
When you start adding this overload, you will see that you need to change the entire calling stack - `AuthorizationError`, `AddValidationError` and so on. You may try. When I worked on #470, I decided to make a change at the lower level.

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/pull/2407